release/cloudbuild_ci.yaml

```diff
@@ -54,13 +54,13 @@ steps:
       - storage
       - cp
       - output/trusted_os.elf
-      - gs://${_TRUSTED_OS_BUCKET}/${_TEST_TAG_NAME}/trusted_os.elf
+      - gs://${_FIRMWARE_BUCKET}/${_FIRMWARE_COMPONENT}/${_TEST_TAG_NAME}/firmware.elf
```

Comment:
I wonder if it might be better to keep the filename as it was?
Reduces the chances of confusion if anyone is digging around/doing manual things?

Reply:
Ok, sounds good. I changed it since we talked about minimizing the number of parameters, but I have reverted it now.

Reply:
Cool, ta.
Just to explain my line of thinking a bit more here - if you look at it a bit sideways, I feel like these "don't really count" as extra params as they're really just static strings selected by the `manifest.firmware_type` param - and we're already doing a `switch` on that param to figure out which directory to look in so we get to define these filenames for free :)